src/modules/launcher/PowerLauncher/ViewModel/RelayCommand.cs

```diff
@@ -17,7 +17,11 @@ public virtual bool CanExecute(object parameter)
             return true;
         }
 
-        public event EventHandler CanExecuteChanged;
+        public event EventHandler CanExecuteChanged
+        {
+            add { }
+            remove { }
```

Comment:
[clarification] Why did we have to add this?

Reply:
This is because this event is never used. So an error is thrown. The following article on MSDN provides a way to cleanly suppress this error: https://docs.microsoft.com/en-us/archive/blogs/trevor/c-warning-cs0067-the-event-event-is-never-used

Reply:
Sounds good. Thanks!